test/ReactSetStateTestComponent.js

```diff
@@ -35,43 +35,43 @@ class ReactSetStateTestComponent extends React.Component {
     _counter = newCount;
   }
 
-  recordLifecyleCall(name) {
+  recordLifecycleCall(name) {
```

Comment:
lol, wow nice fix.

Reply:
Haha thank you! I was looking for typos in my own code and ended up discovering that one.